js/home.js

```diff
@@ -6,6 +6,13 @@ window.onload = function() {
   // display info message if the user is coming for the first time
   if (wkUserData.userPublicKey === undefined || wkUserData.userPublicKey == ""){
     document.querySelector(".info").style.display = 'inline';
+    if (window.location.href.includes('wanikani.com/settings/account')) {
+      alert("Please enter your API Key.")
+    } else {
+      alert("Redirecting to WaniKani...")
+      window.open('https://www.wanikani.com/settings/account/#public-api-key',
```

Comment:
It's a good intention, but too much intrusive :) People don't want to be redirected to a new page without their consent. However you could add a link on the ["API Key" text](https://github.com/naei/wanikani-companion/blob/master/html/settings.html#L29), which will redirect the user to the relevant URL on clicking on it.